web/source/osk/oskKey.ts

```diff
@@ -299,13 +300,28 @@ namespace com.keyman.osk {
       return metrics;
     }
 
-    getIdealFontSize(osk: VisualKeyboard, style: {height?: string, fontFamily?: string, fontSize: string}): string {
-      // Recompute the new width for use in autoscaling calculations below, just in case.
-      let emScale = osk.getKeyEmFontSize();
+    getIdealFontSize(vkbd: VisualKeyboard, style: {height?: string, fontFamily?: string, fontSize: string}): string {
+      let buttonStyle = getComputedStyle(this.btn);
```

Comment:
End goal:  once we can guarantee that `refreshLayout` will always be called at least once after the corresponding `VisualKeyboard` has been added to the DOM, we should be able to drop all parameters for the method and simply rely on `getComputedStyle`.

That guarantee's currently hardest to fulfill for keyboard documentation, since it's built in a state fully detached from the document.  Nothing a temporary `MutationObserver` or `window.setInterval` couldn't 'fix' down the road.